rabbit-escape-ui-android/app/src/main/java/rabbitescape/ui/android/AndroidCanvas.java

```diff
@@ -40,6 +43,24 @@ public void drawLine(
         float startX, float startY, float stopX, float stopY, AndroidPaint paint
     )
     {
-        this.canvas.drawLine( startX, startY, stopX, stopY, paint.paint );
+        this.canvas.drawLine(startX, startY, stopX, stopY, paint.paint);
+    }
+
+    @Override
+    public void drawFilledPoly( int[] xs, int[] ys, AndroidPaint paint )
```

Comment:
These `androidlike` classes are supposed to be a wafer thin wrapper over the Android APIs, so unless there's a really good reason for it, I'd prefer if you added a `drawPath` method here, and made an `androidlike.Path` interface.  I don't particularly like this interface, but I am keen that these classes contain no complexity at all.  We don't want to make a new drawing API here - we want to simulate the Android one for Swing, and wrap the Android one for Android untouched.